File: api/ohara/src/main/java/br/com/ohara/model/Lista.java
package br.com.ohara.model;

import java.util.List;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import lombok.Data;

@Data
public class Lista {
	private Long idLista;
	
	@NotNull 
	private Usuario usuario;

	@NotNull
	@NotBlank(message = "Titulo é obrigatório")
	private String titulo;
	
	private List<Manga> mangas;
}

File: sistemas/mobile/ohara/src/app/model/lista.ts
import { Manga } from "./manga";
import { Usuario } from "./usuario";

export class Lista {
    idLista: number | null;
    titulo: string;
    mangas: Manga[];
    usuario: Usuario;

    constructor() {
        this.idLista = null;
        this.titulo = "";
        this.mangas = [];
        this.usuario = new Usuario();
    }
}
